dependencies.gradle

```diff
@@ -1,12 +1,14 @@
 project.ext.versions = [
     checkstyle: '7.6.1',
-    atlas: '5.3.6',
+    atlas: '5.5.0',
     commons:'2.6',
-    atlas_generator: '4.2.0',
+    atlas_generator: '4.2.3',
+    guava: '27.0.1-jre',
```

Comment:
Once this depends on an Atlas that contains https://github.com/osmlab/atlas/pull/374 we can remove guava from here.